scripts/gen.py

```diff
@@ -97,6 +103,20 @@ def namedtuple_with_defaults(typename, field_names, default_values=()):
     # XLA/TPU functions
 ]
 
+_FN_OUT = {
+    'arange_out(Tensor, Scalar, Scalar, Scalar) -> Tensor':
```

Comment:
It's just these three for now, right?

Reply:
This is an initial PR.
Then we have two options.
We manually feed the list there, or we have a global whitelist below, with few overrides above (like arange, which needs options feeding).
I am not sure ATM how many _out() requires special treatment.